### PR TITLE
Still allow `main` and the last version as active release branch

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind ci

#### What this PR does / why we need it:
We need to allow `main` as well as the latest release branch to be able to build prereleases for it.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
/unhold

PTAL @kwilczynski

File: scripts/reconcile [hunk 1]
```diff
@@ -23,22 +23,27 @@ PACKAGES=$(osc se --package cri-o)
 PROJECT_PREFIX=isv:kubernetes:addons:cri-o
 PROJECT_PRERELEASE_PREFIX=$PROJECT_PREFIX:prerelease
 BUILD_PROJECT_SUFFIX=:build/cri-o
+BRANCH_MAIN=main
 
-ACTIVE_VERSIONS=$(curl_retry https://raw.githubusercontent.com/cri-o/cri-o/refs/heads/main/internal/version/version.go | sed -En "s/.*ReleaseMinorVersions.*\{(.*)\}/\1/p")
+VERSION_URL=https://raw.githubusercontent.com/cri-o/cri-o/refs/heads/$BRANCH_MAIN/internal/version/version.go
```
Comment (kwilczynski) [inline scripts/reconcile:28]:
```suggestion
VERSION_URL=https://raw.githubusercontent.com/cri-o/cri-o/refs/heads/${BRANCH_MAIN}/internal/version/version.go
```

Comment (kwilczynski) [reply scripts/reconcile:28]: The line wrapping is horrible here for some reason.

Comment (saschagrunert) [reply scripts/reconcile]: Fixed

**Pull request merged**